cg/meta/workflow/nf_analysis.py

```diff
@@ -235,6 +300,57 @@ def write_trailblazer_config(self, case_id: str, tower_id: str) -> None:
             file_path=config_path,
         )
 
+    def create_sample_sheet(
+        self,
+        case_id: str,
+        dry_run: bool,
+    ):
+        """Create sample sheet for a case."""
+        sample_sheet_content: list[list[Any]] = self.get_sample_sheet_content(case_id=case_id)
+        if not dry_run:
+            self.write_sample_sheet(
+                content=sample_sheet_content,
+                file_path=self.get_sample_sheet_path(case_id=case_id),
+                header=self.sample_sheet_headers,
+            )
+
+    def create_params_file(
+        self,
+        case_id: str,
+        dry_run: bool,
+    ):
+        """Create parameters file for a case."""
+        LOG.debug("Getting parameters information")
+        workflow_parameters: dict | None = self.get_workflow_parameters(case_id=case_id).dict()
+        if self._append_params_to_nextflow_config:
+            workflow_parameters = None
```

Comment:
is this needed due to that different nf_workflows have diverse behaviour regarding these files?

Reply:
that's correct! this will be gone in the near future once all workflows are standardised